pennylane/transforms/qfunc_transforms.py

```diff
@@ -352,15 +333,15 @@ def circuit(param, transform_weights):
 
             def transform(old_qfunc, params):
                 def new_qfunc(*args, **kwargs):
-                    # 1. extract the tape from the old qfunc, being
+                    # 1. extract the QuantumScript from the old qfunc, being
                     # careful *not* to have it queued.
-                    tape = make_tape(old_qfunc)(*args, **kwargs)
+                    qscript = make_qscript(old_qfunc)(*args, **kwargs)
 
-                    # 2. transform the tape
-                    new_tape = tape_transform(tape, params)
+                    # 2. transform the qscript
+                    new_script = tape_transform(qscript, params)
 
-                    # 3. queue the *new* tape to the active queuing context
-                    new_tape.queue()
+                    # 3. queue the *new* qscript to the active queuing context
+                    new_script.queue()

```

Comment:
Given neither `QuantumTape` nor `QuantumScript` have a `queue` method, maybe we need a better docstring description.

Reply:
probably, but maybe not now. It wouldn't have worked before either. We do say before and after this snippet that it's pseudocode but we could simply not have it at all 👍